md/taxonomy.md

```diff
@@ -249,6 +249,8 @@
     * [Connectors execution](connectors-execution.md)
   * [Living application](_living-application.md)
     * [Customize living application layout](customize-living-application-layout.md)
+    * [Customize resource layouts](customize-resource-layouts.md)
+    * [Bonita Layout](bonita-layout.md)
```

Comment:
Maybe Bonita Layout should be the first page in this section

Reply:
done